.github/workflows/setup-node-and-install.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup Node and Install Dependencies
```

Comment:
Is this a duplicate of the custom action `setup-node-and-install`? Do we need both?